.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "3.4"
+  - "3.5"
+# command to install dependencies
+install: "pip install -r requirements.txt"
+# command to run tests
+script: py.test
```

Comment:
You can write simpler tests with pytest. It also can run unittests, and has an amazing feature called [funcargs/fixtures](http://pytest.org/latest/fixture.html#fixture).